docker-orchestration/sample-platform-python/install.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+#
+# Installer for the CCExtractor sample platform Docker
+#
+# More information can be found on:
+# https://github.com/CCExtractor/sample-platform
+#
+dir=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+root_dir=$(cd "${dir}"/../ && pwd)
+clear
+date=$(date +%Y-%m-%d-%H-%M)
+
+echo "Welcome to the CCExtractor platform installer for docker!"
+echo ""
+echo "-------------------------------"
+echo "|   Installing dependencies   |"
+echo "-------------------------------"
+echo ""
+echo "* Updating package list"
+apt-get update
+echo "* Installing nginx, python, pip, kvm, libvirt and virt-manager"
+apt-get -q -y install nginx python python-dev python3-libvirt libxslt1-dev libxml2-dev python-pip qemu-kvm libvirt-dev virt-manager mediainfo
+echo "* Update pip, setuptools and wheel"
+python -m pip install --upgrade pip setuptools wheel
+echo "* Installing pip dependencies"
+pip install -r "${root_dir}/requirements.txt"
+
+echo "Setting up the directories.."
+mkdir -p "${sample_repository}"
+mkdir -p "${sample_repository}/ci-tests"
+mkdir -p "${sample_repository}/unsafe-ccextractor"
+mkdir -p "${sample_repository}/TempFiles"
+mkdir -p "${sample_repository}/LogFiles"
+mkdir -p "${sample_repository}/TestResults"
+mkdir -p "${sample_repository}/TestFiles"
+mkdir -p "${sample_repository}/TestFiles/media"
+mkdir -p "${sample_repository}/QueuedFiles"
+
+config_db_uri="mysql+pymysql://${db_user}:${db_user_password}@localhost:3306/${db_name}"
+
+echo "-------------------------------"
+echo "|      Finalizing install     |"
+echo "-------------------------------"
+
+echo "* Generating config file"
+echo "# Auto-generated configuration by install.sh
```

Comment:
I'd move this to outside of Docker (the generation) and then pass it in through a volume. Ansible can create this for example.

Reply:
Refer to https://github.com/canihavesomecoffee/sample-platform-ansible/blob/master/configure_platform.yaml, where this config file generation is already handled.

If we pass that generated file through a volume, we can drop a whole bunch of code here and in the docker file (the args IIRC)

Reply:
fixed.